com.orangedatamining.unofficial.metainfo.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.orangedatamining.unofficial</id>
+
+  <name>Orange</name>
```

Comment:
`unofficial` isn't the app name. The app id isn't acceptable either.

Reply:
Roger that, will fix

Reply:
changed to com.github.theoden8.orange-unofficial